plugins/redfish/fu-redfish-plugin.c

```diff
@@ -126,6 +126,16 @@ fu_redfish_plugin_coldplug(FuPlugin *plugin, FuProgress *progress, GError **erro
 	return TRUE;
 }
 
+gboolean
+fu_redfish_plugin_set_credentials(FuPlugin *plugin, const gchar *username, const gchar *password)
```

Comment:
you may as well make this void I think

Reply:
Ah; you're right.  I think it should still be void though, and you should probably leave a comment that it's only intended for self tests.